polkadot/node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -1351,6 +1351,12 @@ impl Initialized {
 				}
 			}
 			for validator_index in new_state.votes().invalid.keys() {
+				gum::trace!(
```

Comment:
Should happen rarely enough to justify "debug"?

Reply:
Okay, let's bump it to DEBUG